index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>Apple</title>
+	<link rel="stylesheet" href="./stylesheets/reset.css">
+	<link rel="stylesheet" href="./stylesheets/main.css">
+  </head>
+
+  <body>
+
+  	<div class="main-container flex-v">
+
+  		<div class="header">
+  			<div class="nav-bar">
+	  			<ul>
+	  				<li><a href="#">
+	  						<img class="logo-img" src="./images/apple-logo.png" alt="">
+	  					</a>
+	  				</li>
+	  				<li><a href="#"><span >Store</span></a></li>
+	  				<li><a href="#"><span>Mac</span></a></li>
+	  				<li><a href="#"><span>iPod</span></a></li>
+	  				<li><a href="#"><span>iPhone</span></a></li>
+	  				<li><a href="#"><span>iPad</span></a></li>
+	  				<li><a href="#"><span>iTunes</span></a></li>
+	  				<li><a href="#"><span>Support</span></a></li>
+	  				<li class="nav-item"><input type="text"><i class="fas fa-search"></i></li>
+	  			</ul>
+	  		</div>
+
+
+			<div class="text-links">
+				<a href="#" class="verse-text common-text"><p>What will your verse be?</p></a>
+
+				<a href="#" class="share-text common-text">
+					<p>Everyone has something to share. See how people are using</p> 
+					<p>iPad in their unique ways to add to the world’s story.</p>
+				</a>
+
+				<a href="#" class="watch-text common-text">
+					<p>Watch the “Your Verse” film and explore the stories <i class="fas fa-play-circle"></i></p>
+				</a>
+			</div>
+
+  		</div>
+
+  		<div class="content">
+  			<ul class="flex-h">
+  				<li>
+  					<a href="#">
+  						<img src="./images/iphone5.jpg" alt="iPhone 5s. Forward thinking.">
+  					</a>
+  				</li>
+
+  				<li>
+  					<a href="#">
+  						<img src="./images/iphone5c.jpg" alt="iPhone 5c. For the colorful.">
+  					</a>
+  				</li>
+
+  				<li>
+  					<a href="#">
+  						<img src="./images/mac-birthday.jpg" alt="Happy Birthday, Mac. Explore 30 years of innovation on the Mac.">
+  					</a>
+  				</li>
+
+  				<li>
+  					<a href="#">
+  						<img src="./images/supplier-responsibility.jpg" alt="Supplier Responsibility The latest results of our worldwide efforts.">
+  					</a>
```

Comment:
[ ] **Required** -These shouldn't be images as this is a project focused on backgrounds and gradients kindly ensure you make these backgrounds instead.